### PR TITLE
Release/satingest.v4.1.0

### PR DESCRIPTION
**Pull request by ShelleyMelchior-NOAA**
Please review merge of `release/satingest.v1.4.0` into `develop`. `release/satingest.v4.1.0` is what NCO is running in para on dogwood and cactus. The para has been running successfully for quite some time, so it is time to merge into `develop`.

### CODE REVIEW

**Review by SteveStegall-NOAA — APPROVED** [review]

File: fix/aerosol [hunk 1]
```diff
@@ -35,6 +35,10 @@ else
    PROC_FAM_group2=" "
 fi
 
+# No longer needed for WCOSS2 because downstream application,
```
Comment (ilianagenkova) [inline fix/aerosol:38]: @ShelleyMelchior-NOAA , do you mean we could remove lines 42-53 and fix/aerosol.hms_smoke_prelim ?

File: fix/aerosol [hunk 2]
```diff
@@ -47,6 +51,10 @@ else
    PROC_FAM_group3=" "
 fi
 
+# No longer needed for WCOSS2 because downstream application,
```
Comment (ilianagenkova) [inline fix/aerosol:54]: Save question as before, do you imply we remove lines 58-87 along with fix/aerosol.hms_smoke_final from the satingest s/w suite?

**Review by ilianagenkova — CHANGES_REQUESTED** [review]
@ShelleyMelchior-NOAA ,
I left a couple of comments/questions , just to clarify what we need to clean up w.r.t. hms data.

I am not a satingest expert, so as sanity check I'll try compiling the code  before approving/merging this PR. I will try to wrap it up in a day or two. thanks!

**Review by ilianagenkova — APPROVED** [review]

**Comment by ilianagenkova** [comment]
@SudhirNadiga-NOAA  I made you a reviewer just for awareness.

All, I found GitHub Issue #8 (thanks, @ShelleyMelchior-NOAA !). It documents well what needs to be cleaned in the satingest code shortly after WCOSS2-golive . I approved this PR, but will keep the GitHub issue open, to remind us what we need to do later on. This is a regular practice on GitHub - open issues document what is still in the works.

**Comment by ShelleyMelchior-NOAA** [comment]
Excellent -- thank you @ilianagenkova

**Pull request merged**